instrumentation/kamon-netty/src/main/scala/kamon/netty/instrumentation/mixin/HttpMessageMixin.scala

```diff
@@ -0,0 +1,26 @@
+package kamon.netty.instrumentation.mixin
```

Comment:
This class should be removed and use `HasContext` from instrumentation-common instead